Wire-iOS/Sources/UserInterface/UserProfile/Details/ProfileDetailsContentController.swift

```diff
@@ -107,8 +115,9 @@ class ProfileDetailsContentController: NSObject, UITableViewDataSource, UITableV
     
     /// Updates the content for the current configuration.
     private func updateContent() {
-        switch conversation?.conversationType {
-        case .group?:
+        switch (conversation?.conversationType, context) {
```

Comment:
```suggestion
        switch conversation?.conversationType ?? .oneOnOne {
```

We can achieve the same thing without the context if we default to `.oneOnOne` if the conversation is missing.

Reply:
I agree this would be a better solution. We could update the existing pattern matching to `case .oneOnOne?, .none`.